radicle-node/src/service.rs

```diff
@@ -436,6 +440,9 @@ where
             self.reactor.wakeup(PRUNE_INTERVAL);
             self.last_prune = now;
         }
+
+        // Always check whether there are persistent peers that need reconnecting.
+        self.maintain_persistent();
```

Comment:
Would better naming the method remove the need for the comment?

Reply:
Don't think so. I left the comment because it seems "random" for this one method to be called down here after all the `if{}` statements, and I want to explain why we're calling it here.